core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java

```diff
@@ -34,27 +33,35 @@ public class Sep10Service {
 
   public Sep10Service(
       AppConfig appConfig, Sep10Config sep10Config, Horizon horizon, JwtService jwtService) {
+    infoF("Creating Sep10Service");
+    infoConfig("appConfig:", appConfig, AppConfig.class);
     this.appConfig = appConfig;
+
+    infoConfig("sep10Config:", sep10Config, Sep10Config.class);
     this.sep10Config = sep10Config;
+
     this.horizon = horizon;
     this.jwtService = jwtService;
     this.serverAccountId = KeyPair.fromSecretSeed(sep10Config.getSigningSeed()).getAccountId();
   }
 
   public ChallengeResponse createChallenge(ChallengeRequest challengeRequest) throws SepException {
+    info("Creating challenge");
     //
     // Validations
     //
     if (challengeRequest.getHomeDomain() == null) {
+      infoF("home_domain is not specified. Use {}", sep10Config.getHomeDomain());
```

Comment:
What's the reason for using `info` instead of `debug` here (and in other parts of the project)? It's more intuitive to me that debug is used when we're walking through the code, while Info is something more high-level. For instance:

```text
INFO:  Creating challenge
DEBUG: home domain is not specified
DEBUG: fetching account from Stellar network
ERROR: something went wrong when fetching the Stellar account.
```

Not a blocking change, nor it's written in stone... it's a concept that we will eventually reach a common definition as we work together.